UI/window-basic-main.cpp

```diff
@@ -4373,6 +4373,11 @@ void OBSBasic::ResetUI()
 		ui->previewLayout->setDirection(QBoxLayout::LeftToRight);
 
 	UpdatePreviewProgramIndicators();
+
+	bool hideTransitionWidget = config_get_bool(
+		App()->GlobalConfig(), "BasicWindow", "hideTransitionWidget");
```

Comment:
Mismatch with others.
```suggestion
		App()->GlobalConfig(), "BasicWindow", "HideTransitionWidget");
```